vendor/github.com/blang/semver/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License
```

Comment:
It was not found in the [Travis build(]https://travis-ci.org/kubernetes-incubator/cri-containerd/jobs/225271496#L222)